src/main/scala/engine/SimulationConstants.scala

```diff
@@ -18,14 +18,17 @@ object SimulationConstants {
   val GENERATION_END: FiniteDuration = 12000 millis
 
   /**Maximum number of alive bunnies in the world*/
-  val MAX_BUNNIES_NUMBER = 1000
+  val MAX_BUNNIES_NUMBER: Int = 1000
```

Comment:
```suggestion
  val MAX_ALIVE_BUNNIES: Int = 1000
```
il nome così a me suona più esplicativo, in modo da poter anche togliere l'eventuale commento (così diventa tutto più autocontenuto)